ocs_ci/ocs/resources/storage_cluster.py

```diff
@@ -378,6 +378,9 @@ def ocs_install_verification(
         check_fips_enabled()
     if config.ENV_DATA.get("encryption_at_rest"):
         osd_encryption_verification()
+        if config.DEPLOYMENT.get("kms_deployment"):
+            if config.ENV_DATA.get("KMS_PROVIDER") == constants.VAULT_KMS_PROVIDER:
```

Comment:
Should we have a generic `post_deploy_verification` method in the `KMS` class that calls the provider-specific method based on `config.ENV_DATA.get("KMS_PROVIDER")`? This way we don't need to check for each provider type in this code (code that really doesn't care about WHICH kms provider we are using).

Reply:
I guess you meant like this. Makes sense, I will do it.
```python
if config.DEPLOYMENT.get("kms_deployment"):
    kms = KMS.get_kms_deployment()
    kms.post_deploy_verification()
```